test/README.md

```diff
@@ -0,0 +1,18 @@
+# Running Tests
+
+To run the tests, perform the following steps:
+
+1) Startup a fresh iRODS server.
+2) Startup the Testing keycloak image.
+3) Startup the iRODS HTTP API server.
+4) Ensure the HTTP API can communicate with the iRODS server and the keycloak image.
+5) Configure the [config.py](config.py) to the appropriate test configuration.
+6) Install `pytest` for the OpenID tests.
+7) Run `pytest` to run all tests. If desired, run `test_irods_http_api.py` directly if OpenID tests are not required.
+
+## Testing Keycloak Image
```

Comment:
How does one launch the build/launch the image?

Reply:
Will add to image section. :+1:

Reply:
Added documentation to build and launch the image.